### ISSUE TITLE
Contribution Guidelines needed to guide new contributors through copyright issues

### ISSUE THREAD

**Issue by asdofindia** [mozilla/OpenDesign#224]
## Goal: 
Contribution guidelines for open design

## Info: 
It will be used for guiding new contributors to the open design community to avoid copyright issues in the designs that they create.

## Style Information: 
It will look like http://www.howdesign.com/featured/graphic-design-copyright-laws-inspiration-vs-infringement/ https://opensource.guide/legal/ and https://www.business.com/articles/the-copyright-question-the-difference-between-inspiration-and-infringement/ put together

## Deadline: 
Whenever

## Tag: 
[Volunteer needed]

**Comment by pransh15** [comment]
Hello!

Do you have the content already?

**Comment by asdofindia** [comment]
@pransh15 erm, content is there all over the web. (Like the links I put in the issue).

**Comment by kindavishal** [comment]
Is this related to design or just updating the repo with content for Contribution guidelines?

**Comment by asdofindia** [comment]
It is a meta issue, not a request for a design

**Comment by kindavishal** [comment]
I understand, this is indeed important. We can work on writing a template for this.
Tagging @elioqoshi @yofiesetiawan @barrosgeraldo @pransh15 for suggestions on this.

**Comment by kindavishal** [comment]
Please tag more people who you think can be helpful in this.

**Comment by Ryuno-Ki** [comment]
We're basically talking about adding a [CONTRIBUTING.md file](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors) here, aren't we?
(Some recommendations by GitHub at the end of the article).

**Comment by kindavishal** [comment]
Yes, we're talking about that, but more about what goes into the file, the content basically

**Comment by elioqoshi** [comment]
Sorry for missing this.

The idea was in the beginning that all content here would be licensed under CC-BY 4.0 for external usage and no need for attribution for usage within Mozilla projects. 

My suggestion for attribution would be to refer to it as "Mozilla Open Design" instead of individual contributors and list the list of core contributors in the ReadMe and/or Contributors.md

Some house keeping is long overdue definitely. I will try to help with that once I get a chance.

**Comment by asdofindia** [comment]
Elio, I will add some context. There is very little awareness about the concept of copyright among the people of India. This is certainly applicable to "graphic designers" as well. Many a times, designs created are blatantly violating others' rights. I have seen this happening within Mozilla community. The need for guidelines is here. To avoid content of questionable ownership being contributed to this repository

**Comment by elioqoshi** [comment]
@asdofindia Right, I didn't touch on this point. Unfortunately we cannot check ourselves if something is violating any copyright but we can definitely create guidelines to ensure people are aware at least. Generally speaking, since we are offering designs here for Mozilla projects without attribution, we would only be able to use CC0 Public Domain resources for the created artwork here. Anything which is not Public Domain or where the contributor holds the rights of, is not compatible with the licensing of this repo.

**Comment by kindavishal** [comment]
Adding my 2 cents here, I think if we describe the guidelines as mentioned in [Design Mozilla](https://mozilla.design) since most of the work on this repo mainly revolves around them can be a point.
